server/tokens.go

```diff
@@ -131,7 +188,10 @@ func (c *Server) blindedTokenRedeemHandler(w http.ResponseWriter, r *http.Reques
 				Message: "Could not mark token redemption",
 				Code:    http.StatusInternalServerError,
 			}
-
+		}
+		err := json.NewEncoder(w).Encode(blindedTokenRedeemResponse{verifiedCohort})
```

Comment:
Will the addition of this response body cause any clients to fail?  Do we need to break this into a blindedTokenRedeem v2 since we changed the response structure?

Reply:
If that is best practice, then yes, I'll change it. As far as I can see in the tests, the way this response is being processed is simply by checking that the response is not an error. Let me know what you think

Reply:
Discussed with @amirsaber , the current ads server only parses the body when the response is not 200, therefore this won't cause clients of the API (aka the ads server) to fail.